content/reference/host-runtime/actors.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Actors"
+date: 2018-12-29T11:02:05+06:00
+weight: 4
+draft: false
+---
+
+An **_actor_** is the smallest unit of deployable, portable compute within the wasmCloud ecosystem. Actors are small WebAssembly modules that can handle messages delivered to them by the host runtime and can invoke functions on [capability providers](../capabilities), provided they have been [granted the appropriate privileges](../security).
+
+### Single-Threaded
+
+_Concurrency is hard_. Even when we get concurrency correct, it's still hard. Building systems that work properly either through multi-threading or through so-called "green threads" or "coroutines" is difficult. Concurency introduces friction in writing new code, maintaining old code, troubleshooting, and routinely wreaks havoc on production systems.
```

Comment:
Typo here with one `r` in Concurrency

Reply:
Clearly the single-r spelling is the southwest Gondor spelling.